frontend/src/metabase/visualizations/lib/settings/column.js

```diff
@@ -220,9 +220,10 @@ export const DATE_COLUMN_SETTINGS = {
     getHidden: ({ unit }, settings) => !/\//.test(settings["date_style"] || ""),
   },
   date_abbreviate: {
-    title: t`Abbreviate names of days and months`,
+    title: t`Abbreviate days and months`,
     widget: "toggle",
     default: false,
+    inline: true,
```

Comment:
This is interesting! without this option, the setting doesn't render at all. What does this option actually do?

Reply:
It just puts the label and the toggle on the same line. Without it, it renders like this: 
![image](https://user-images.githubusercontent.com/1328979/187921171-69e2ed05-78d7-4b6f-bf64-4754ae424478.png)

Reply:
Interesting... That isn't what I experienced, but I'll check again tomorrow.